src/backend/mailbox_manager/tests/api/mail_domain/test_api_mail_domains_create.py

```diff
@@ -79,5 +97,93 @@ def test_api_mail_domains__create_authenticated():
 
     # a new domain with status "pending" is created and authenticated user is the owner
     assert domain.status == enums.MailDomainStatusChoices.PENDING
-    assert domain.name == "mydomain.com"
+    assert domain.name == domain_name
     assert domain.accesses.filter(role="owner", user=user).exists()
+
+
+## SYNC TO DIMAIL
+@mock.patch.object(Logger, "info")
```

Comment:
Would you consider using `caplog`? ^^

Reply:
We could, although i'd rather discuss/do it in a dedicated PR.

I'm not sold because I'm not sure makes it easier to read. What would be the gain of using caplog, compared to mock ?

Reply:
I finally switched to caplog ! I'll have to harmonize uses of caplog on the other tests where we used mock_error